archive_db.py

```diff
@@ -0,0 +1,58 @@
+# standard libraries
+import json, logging, os
+from datetime import datetime
+
+# third-party libraries
+import pandas as pd
+from sqlalchemy import create_engine
+
+
+# Settings and global variables
+
+logger = logging.getLogger(__name__)
+logging.basicConfig()
+
+try:
+    with open('config/env.json') as f:
+        ENV = json.load(f)
+except FileNotFoundError as fnfe:
+    logger.error('No configuration file was found; add env.json to the config directory.')
+
+logger.setLevel(ENV.get('LOG_LEVEL', 'DEBUG'))
+
+OUT_DIR = ENV.get('OUT_DIR', 'data')
```

Comment:
make sure the OUT_DIR exists before writing csv files?

Reply:
This should be addressed by my latest commits. Let me know if you have any questions!